django_ipam/api/views.py

```diff
@@ -0,0 +1,16 @@
+import swapper
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+Subnet = swapper.load_model('django_ipam', 'Subnet')
+
+
+@api_view(['GET'])
+def iprequest(request):
+    subnet_id = request.query_params.get('subnet_id')
+    subnet = Subnet.objects.get(pk=subnet_id)
+    ipaddress_set = [ipaddr.ip_address for ipaddr in subnet.ipaddress_set.all()]
+    for host in subnet.subnet.hosts():
+        if str(host) not in ipaddress_set:
+            return Response(str(host))
+    return Response('No IP address is available')
```

Comment:
I'd use `return Response(None)` which would be easier to parse in client implementations, we will have to document this